src/Omega_h_adios2.hpp

```diff
@@ -14,6 +14,7 @@ void write(filesystem::path const& path, Mesh *mesh, std::string pref="");
 
 Mesh read(filesystem::path const& path, Library* lib, std::string pref="");
 
+void write_mesh(adios2::IO &io, adios2::Engine & writer, Mesh* mesh, std::string pref);
```

Comment:
This seems like it should be part of another PR.

Reply:
I agree, this should be a separate pull request.